### ISSUE TITLE
export/md: info boxes removed

### ISSUE THREAD

**Issue by nasearle** [googlecodelabs/tools#45]
When using the tool to export a Google doc (e.g. this [Workbox lab](https://docs.google.com/document/d/1GZsXkUCKZ4xAsVxeAajOdnbvVFa06syvUV9Rj31wQgk/edit#)) as markdown (with `claat export -f md 1GZsXkUCKZ4xAsVxeAajOdnbvVFa06syvUV9Rj31wQgk`), the tool is removing info boxes entirely. The info boxes are styled according to the [Codelab Authoring Guide](https://docs.google.com/document/d/1VjyoX4DqeTMB6gr63ss_hAByRpzELkXuIaIrioMisBo/edit#heading=h.33hkuldw88lv).

**Comment by x1ddos** [comment]
@nasearle could you make a copy of that doc of yours and make it readable by "whoever has the link" please? I don't have access.

**Comment by nasearle** [comment]
I just fixed this issue for my own team. The `render/md.go` file was missing an infobox function that I copied from `render/html.go` and tweaked for our purposes. The function looks like this:

```
func (mw *mdWriter) infobox(n *types.InfoboxNode) {
  mw.newBlock()
  mw.writeString(`<div class="`)
  mw.writeString(string(n.Kind))
  mw.writeString(`">`)
  mw.write(n.Content.Nodes...)
  mw.writeString("</div>")
}
```

And I uncommented the call to `infobox` in the mdWriter function:

```
case *types.InfoboxNode:
  mw.infobox(n)
```

**Comment by x1ddos** [comment]
Ah cool! It's been such a long time I forgot what worked and what didn't.
Thanks for the example.

**Comment by samtstern** [comment]
This is still the case.  Note that with the new md parser we can use the "detail box" syntax.

**Comment by samtstern** [comment]
Implemented a fix for this in #65.  It uses an `aside`, in the future I'd like for a pure markdown solution to work (since we have support for "detail boxes") but I was not able to get it done.
